---
robot_id: $(arg robot_id)
odom_frame: $(arg odom_frame)
robot_frame: $(arg robot_frame)
map_frame: $(arg map_frame)
exit_after_clock: $(arg exit_after_clock)
enable_frontend_output: $(arg enable_frontend_output)
semantic_colormap_file: $(arg semantic_map_path)
enable_lcd: $(arg enable_lcd)
log_path: $(arg log_path)
frontend/objects/bounding_box_type: $(arg bounding_box_type)
frontend/validate_vertices: false
backend/dsg/optimize_on_lc: $(arg optimize_on_lc)
backend/dsg/enable_merge_undos: $(arg enable_merge_undos)
backend/dsg/use_zmq_interface: $(arg use_zmq_interface)
backend/dsg/zmq_send_url: $(arg zmq_send_url)
backend/dsg/zmq_recv_url: $(arg zmq_recv_url)
loop_closure/type: LoopClosureModule
loop_closure/detector/agent/descriptors/vocabulary_path: $(arg vocabulary_path)
